public/bitemp_graph_helper.js

```diff
@@ -144,8 +142,88 @@ function edit(uri) {
   }
 }
 
-function deleteDoc(uri) {
-  if (!uri) { // Not given a valid document uri
+function getDocColl(uri) {
+  console.log('Getting docColl');
+  var docColl = $.ajax({
+    url: "/v1/documents?uri="+uri+"&category=collections&format=json",
+    success: function(data, textStatus) {
+     console.log('got collections: ' + data);
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      console.log('problem');
+    },
+    async: false,
+  });
+ 
+ return JSON.parse(docColl.responseText);
+}
+
+var deleteDoc = function (chart) {
+  var uri = chart.getLogicalURI();
+  var ajax = true;
+  if (!uri) {
+    return;
+  }
+  var collection = getDocColl(uri);
+  var url = '/v1/documents?uri=' + uri + '&temporal-collection='+collection.collections[0];
```

Comment:
there is no guarantee that temporal collection will always be the first one in the response, check this call out to find all temporal collections:
http://pubs.marklogic.com:8011/REST/GET/manage/v2/databases/[id-or-name]/temporal/collections